VimeoNetworking/Sources/Models/Subscription.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  self.swift
+//  Pods
+//
+//  Created by Lim, Jennifer on 2/8/17.
+//
+//
+
+import Foundation
+
+public class Subscription: VIMModelObject
```

Comment:
We probably don't need to explicitly redeclare this or the properties below as `public` if we're inheriting from something that's public.